community/modules/scripts/windows-startup-script/README.md

```diff
@@ -0,0 +1,90 @@
+## Description
+
+This module contains a set of scripts to be used in customizing Windows VMs at
+boot or during image building. Please note that the installation of NVIDIA GPU
+drivers takes, at minimum, 30-60 minutes. It is therefore recommended to build
+a custom image and reuse it as shown below, rather than install GPU drivers at
+boot time.
+
+> NOTE: the output `windows_startup_ps1` must be passed explicitly as shown
+> below when used with Packer modules. This is due to a limitation in the `use`
+> keyword and inputs of type `list` in Packer modules; this does not impact
+> Terraform modules
+
+```yaml
+- group: primary
+  modules:
+  - id: network0
+    source: modules/network/pre-existing-vpc
+
+  - id: windows_startup
+    source: community/modules/scripts/windows-startup-script
+    settings:
+      install_nvidia_solutions: true
```

Comment:
Elsewhere we use `install_nvidia_driver`. The [script](https://github.com/GoogleCloudPlatform/compute-gpu-installation/blob/main/windows/install_gpu_driver.ps1) you reference is also called `install_gpu_driver`. Is there a reason to change terminology?

Reply:
1. I do not want to assume that GPU = NVIDIA (even if ubiquitious)
2. In the other cases, the script only installs drivers and I am seeking to convey that this also installs the CUDA Toolkit (whether or not the script filename is entirely right)

Reply:
I don't see relevance of point 1 to the final naming as you are still using nvidia in the name.
Point 2 seems like a reasonable tradeoff. My preference would still be for `install_nvidia_driver` for API consistency and document what you are actually doing. 

I think you are more likely to have a user get an error because they try `install_nvidia_driver` and it does not work. Than you are to have someone complain that cuda was installed. 

All that said, consider this a nit and use your judgement.

Reply:
1. the point is that I am purposefully using NVIDIA (not "GPU") in the name because it is not the only possible implementation of drivers and software libraries, even for NVIDIA accelerators.
2.  I'll consider `install_nvidia_driver` even though it is doing more than what it is doing elsewhere.

Reply:
Changed to `install_nvidia_driver` as more in spirit of intent even if not precise in details.